docs/development/release_guide.rst

```diff
@@ -29,16 +29,16 @@ Pre-release
 Release
 -------
 
-* Reserve a digital object identifier (DOI) on `Zenodo`_ for version ``0.6.0``.
+* Reserve a digital object identifier (DOI) on Zenodo_ for version ``0.6.0``.
```

Comment:
And this works? Without the `s?

I've been catching up on Star Trek TOS recently and let me just say, fascinating.

Reply:
Yep!  One of the many things I learned but never wanted to while working on #1156.